addons/social_share/models/template_group.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+class CampaignTemplate(models.Model):
+    _inherit = 'mail.render.mixin'
+    _name = 'snshare.template.group'
```

Comment:
Ok so I consider all this will be cleaned / improved in followups. The language field is present on the form view of snshare.template.group, as this is a default form view with model fields. But as you said, I guess templates are not supposed to have their menu / view since they are not really editable yet. -> group view should not be used either.

For reference (the language field)
![image](https://github.com/odoo/odoo/assets/71645720/9711f1de-cd0b-4d25-ab86-235f89bf2add)

Reply:
Ah I see, indeed it's just the default form view so it has all the fields of the model. Strangely I don't see the mixin fields locally though :thinking: